vault/resource_pki_secret_backend_cert.go

```diff
@@ -151,6 +151,12 @@ func pkiSecretBackendCertResource() *schema.Resource {
 				Computed:    true,
 				Description: "The certificate expiration.",
 			},
+			"auto_revoke": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
```

Comment:
Since the default behaviour was not to revoke, it probably makes sense to set the default to `false`
```suggestion
```